### ISSUE TITLE
Add Accessibility Note on CSS Page

### ISSUE THREAD

**Issue by loganfranken** [UCSantaBarbara/webguide#106]
On the [CSS page](http://webguide.ucsb.edu/css/), we have the following:

> As a general rule of thumb, the content of your website should have a logical flow regardless of any styling. In other words, a user who has disabled CSS should still be able to understand your website’s content.

There isn't any explanation as to _why_ you should do this. We should add in some of the text from the [accessibility guide](https://github.com/ucsb-wsg/web-accessibility/blob/master/web-accessibility-guidelines.md#2-do-not-rely-on-color-alone-to-convey-information) to provide context:

> Information conveyed by color alone is inaccessible to individuals who are color blind as well as people who are using a screen reader to convey web information. For instance, instructions such as "click on the green button to continue" should not be used unless the green button is identified in a format accessible to persons with colorblindness or other disabilities which do not allow them to distinguish colors. A better method would be to use an emphasized elment <em></em> or strong element <strong></strong> or mark the item with an \* as well as a color.
